src/cirrus/builtins/functions/update-state/lambda_function.py

```diff
@@ -23,18 +25,10 @@
 SQS_CLIENT = boto3.resource("sqs")
 QUEUE = SQS_CLIENT.Queue(PROCESS_QUEUE_URL)
```

Comment:
You don't need these anymore, right?

Reply:
Correct.

Reply:
I addressed this before reading the back-story.  May need to discuss how that relates here.